### PR TITLE
Add e2e for json import via url parameter.

### PR DESCRIPTION
**Pull request by ioedeveloper**
E2E test for https://github.com/ethereum/remix-project/pull/2054

### CODE REVIEW

File: apps/remix-ide-e2e/src/tests/url.spec.ts [hunk 1]
```diff
@@ -104,6 +104,21 @@ module.exports = {
       .verify.elementPresent('#runs:disabled')
       .click('[for="optimize"')
       .verify.attributeEquals('#runs', 'value', '200')
+  },
+
+  'Should load json files from link passed in remix URL': function (browser: NightwatchBrowser) {
+    browser
+      .url('http://localhost:8080/#optimize=false&runs=200&evmVersion=null&version=soljson-v0.6.12+commit.27d51765.js&url=https://raw.githubusercontent.com/EthVM/evm-source-verification/main/contracts/1/0x011e5846975c6463a8c6337eecf3cbf64e328884/input.json')
```
[the remark this answers is not in the document]
Comment (ioedeveloper) [reply apps/remix-ide-e2e/src/tests/url.spec.ts:111]: If we add this in our e2e folder, there is no way to import it via url params.

[the remark this answers is not in the document]
Comment (yann300) [reply apps/remix-ide-e2e/src/tests/url.spec.ts:111]: we can't use the raw github url? like https://raw.githubusercontent.com/ethereum/remix-project/master/apps/remix-ide-e2e/nightwatch.ts

[the remark this answers is not in the document]
Comment (ioedeveloper) [reply apps/remix-ide-e2e/src/tests/url.spec.ts:111]: We can but the file will need to be uploaded first.

[the remark this answers is not in the document]
Comment (yann300) [reply apps/remix-ide-e2e/src/tests/url.spec.ts:111]: sure we can upload it in another PR.

[the remark this answers is not in the document]
Comment (ioedeveloper) [reply apps/remix-ide-e2e/src/tests/url.spec.ts:111]: Ok, so we keep this url `https://raw.githubusercontent.com/EthVM/evm-source-verification/main/contracts/1/0x011e5846975c6463a8c6337eecf3cbf64e328884/input.json` for this PR?

**Review by yann300 — APPROVED** [review]

File: apps/remix-ide-e2e/src/tests/url.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (yann300) [reply apps/remix-ide-e2e/src/tests/url.spec.ts:111]: sure!

**Pull request merged**